src/scripts/app-controller.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-env browser */
 
 import PushClient from './push-client.js';
-import EncryptionHelperFactory from './encryption/encryption-helper';
+import EncryptionHelperFactory, {EncryptionHelper}
```

Comment:
I think that if you make *neither* of these a default export you can tidy up the import line.